base/client.yaml

```
@@ -2,6 +2,10 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: &app cockroachdb-client
+  annotations:
+    "app.uw.systems/description": "Used to connect and query the cockroachdb databases."
+    "app.uw.systems/repos.cockroachdb-manifests": "https://github.com/utilitywarehouse/cockroachdb-manifests"
+    "app.uw.systems/tier": "tier_4"
```

You probably want to set a language to, since the default is 'Go' which enables some other checks https://wiki.uw.systems/posts/ops-level-nz4v4ka0#h98mm-app-uw-systems-language

```suggestion
    "app.uw.systems/tier": "tier_4"
    "app.uw.systems/language": Bash
```